mkosi.md

```diff
@@ -653,6 +653,11 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   in building or further container import stages.  This option strips
   SELinux context attributes from the resulting tar archive.
 
+`MachineID=`, `--machine-id`
+
+: Set the machine's ID to the specified value. If unused, a random ID will
+be generated using uuid4().
```

Comment:
No need to detail implementation details here. Just mention that a random machine ID will be used while building the image and that the final image will ship without a machine ID.